src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2028,6 +2028,21 @@ Here is the ordered list of rules for the EDF policy:
 
 6. job with earliest submission time.
 
+==== Email notificaions
+
+EDF policy sends 3 types of email notifications:
+
+1. [UNDER CONSTRUCTION] when job deadline is at risk (if we do not start job in next X minutes we will miss its deadline)
+
+2. when job most likely will miss its deadline (we already at the moment that job will miss its dealine, if `JOB_EXEC_TIME` is precise)
```

Comment:
Do you mean that: If the job will likely miss its deadline (i.e if the provided `JOB_EXEC_TIME` exceeds the left time until the deadline)